presses-universitaires-de-rennes-archeologie-et-culture.csl

```diff
@@ -414,14 +414,13 @@
       <key macro="title"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="&#160;; ">
-      <group delimiter="&#160;: ">
-        <group delimiter="&#160;">
-          <text macro="author-citation"/>
-          <text macro="year-date"/>
-        </group>
-        <text variable="locator"/>
-      </group>
-    </layout>
+     <text macro="author-citation" suffix=", "/>
+     <text macro="year-date" suffix=",&#160;"/>
+     <group delimiter="&#160;">
+       <label variable="locator" form="short"/>
+       <text variable="locator"/>
+     </group>
+   </layout>
```

Comment:
Thanks. This still doesn't look quite right, though. Could you try replacing:

```xml
     <text macro="author-citation" suffix=", "/>
     <text macro="year-date" suffix=",&#160;"/>
     <group delimiter="&#160;">
       <label variable="locator" form="short"/>
       <text variable="locator"/>
     </group>
```

by

```xml
     <group delimiter=",&#160;">
       <group delimiter=", ">
         <text macro="author-citation"/>
         <text macro="year-date"/>
       </group>
       <group delimiter="&#160;">
         <label variable="locator" form="short"/>
         <text variable="locator"/>
       </group>
     </group>
```
?